nemo/collections/asr/data/vocabs.py

```diff
@@ -80,7 +94,13 @@ def decode(self, tokens: List[int]) -> str:
 class Chars(Base):
     """Chars vocabulary."""
 
-    def __init__(self, punct=True, spaces=False, apostrophe=True):
+    def __init__(
+        self,
+        punct=True,
+        spaces=False,
+        apostrophe=True,
+        add_blank_at="last_but_one",
```

Comment:
what's that argument for?

Reply:
For using ASR models in NeMo we assume that blank index in vocab is last. For saving back compatibility with previous version of `vocabs.py` I created this argument.

Reply:
i don't endorse this, but ok
better option would be to specify blank id inside CTC loss